network/prober/prober_test.go

```diff
@@ -264,6 +264,54 @@ func TestAsyncMultiple(t *testing.T) {
 	}
 }
 
+func TestWithPathOption(t *testing.T) {
+	path := "/correct/probe/path/"
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		t.Logf("Want: %s, Got: %s\n", path, r.URL.Path)
```

Comment:
got before want? It's logging, so I'm less nitpicky here :)

Reply:
Done